sections/04_study.md

```diff
@@ -134,57 +129,50 @@ exon skipping [@tag:Barash2010_splicing_code @tag:Xiong2011_bayesian]. With the
 advent of deep learning, more complex models were built that provided better
 predictive accuracy [@tag:Xiong2015_splicing_code
 @tag:Jha2017_integrative_models]. Importantly, these new approaches can take in
-not only genomic features, but also tissue identity and CLIP-seq measurements of
-interactions between splicing factors and RNA, which all improve predictive
-accuracy.
-
-The massive improvement seen with deep learning seems to stem from hidden layers
-being able to create new higher-order “features”, whereas earlier approaches
-often assumed independence of features and were unable to generalize.
-Higher-order understanding is especially important in splicing, which depends
-not only on the primary sequence, but also local RNA structure, tissue identity,
-splicing factor binding, and other currently unknown factors — all of which
-interact in complex, incompletely characterized ways. With new tools to
-interpret these meta-features, a major focus of current deep learning research,
-we will soon have the ability to extract a more nuanced biological understanding
-of splicing — perhaps by interrogating informative hidden nodes within neural
-networks that take in tissue type as part of the input, or by building separate
-networks for each tissue type and looking for common versus distinctive nodes
+multiple kinds of epigenomic measurements, as well as tissue identity and RNA
+binding partners of splicing factors. Deep learning is critical in furthering
+these kinds of integrative studies where different data types and inputs
+interact in unpredictable (often nonlinear) ways to create higher-order “features”,
+compared to earlier approaches that often assumed independence of features or
+required extensive human fine-tuning. Moreover, as in gene expression network
+analysis, interrogating the hidden nodes within neural networks will likely
+yield new biological insights into splicing. For instance, tissue-specific
+splicing mechanisms can be inferred by training networks on splicing data from
```

Comment:
`Qin2017_onehot` studies TF binding, not splicing, right?  If so, should this state "mechanisms could be inferred"?  I think the idea is that one could use a model like Qin 2017 for tissue-specific splicing models.  We may even want to point out that Qin 2017 was for TF binding.